### PR TITLE
Various changes in field types doc

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: docs/source/fieldtypes.rst [hunk 1]
```diff
@@ -206,37 +206,24 @@ For instance:
...
+            "type": "annotation",
+            "css": "font-weigth: bold"
+          },
+          {
+            "label": "Picture",
+            "name": "picture",
+            "type": "url",
+            "required": false
           }
         ]
```
Comment (leplatrem) [inline docs/source/fieldtypes.rst:228]: I know you wanted to keep the same example all along, but this was too much to digest as an annotation example :) I simplified it as much as I could...

**Comment by leplatrem** [comment]
No feedback ? Merge ?

File: docs/source/fieldtypes.rst [hunk 2]
```diff
@@ -5,15 +5,17 @@ Daybed field types
...
-* name
-* label
-* hint
-* required
-* type
+* **label**: human readable field label
+* **name**: storage attribute name
+* **type**: kind of value to be stored
+
+* **required**: mandatory attribute (*default: true*)
```
Comment (almet) [inline docs/source/fieldtypes.rst]: `optional, *default: true*`

File: docs/source/fieldtypes.rst [hunk 3]
```diff
@@ -33,14 +35,12 @@ For example, when combining global properties and one of those basic field, it b
...
-        "name": "item",
-        "hint": "Enter an Todo item",
-        "required": true,
+        "label": "Task item",
+        "name": "task",
         "type": "string"
     }
 
-You just have to choose the right basic type.
+You just have to choose the a type among those available.
```
Comment (almet) [inline docs/source/fieldtypes.rst]: "the a" should be "a type".

File: docs/source/fieldtypes.rst [hunk 4]
```diff
@@ -206,37 +206,24 @@ For instance:
 
     {"definition":
       {
-        "title": "Movies",
-        "description": "List of movies I like.",
+        "title": "Wishlist",
+        "description": "List of gifts you desire",
```
Comment (almet) [inline docs/source/fieldtypes.rst]: consumerism!

File: docs/source/fieldtypes.rst [hunk 5]
```diff
@@ -344,8 +373,27 @@ of fields or the definition of the specified model
     }
 
 
-Relations
----------
+If ``item`` is not specified, the list items can be anything:
```
Comment (almet) [inline docs/source/fieldtypes.rst]: - `e.g. no validation will be done on them`.

File: docs/source/fieldtypes.rst [hunk 6]
```diff
@@ -373,8 +421,9 @@ Relations
       "label": "Main character"
     }
 
-Geometries
-----------
+
+Geometry types
```
Comment (almet) [inline docs/source/fieldtypes.rst]: should it be "Geometric types"?

**Pull request merged**